docs/content/sips/016-component-imports.md

```diff
@@ -0,0 +1,53 @@
+title = "SIP 016 - Spin Component Imports" 
+template = "main" 
+date = "2023-09-22T16:27:43Z"
+---
+
+Summary: A proposal to introduce component imports to the Spin 2.0 manifest.
+
+Owner: <brian.hardock@fermyon.com>
+
+Created: Sep 19, 2023
+
+## Background / Goals
+The Spin 2.0 framework will support executing preview2 WebAssembly components. These components will include `import` and `export` statements. This manifest proposal is to enable Spin developers the ability to expressly configure how to satisfy the imports of their components. Furthermore, developers should be enabled to use the exports of a component to satisfy the imports of another.
```

Comment:
Can we add a link to components for those unfamiliar?

How do we concisely represent where the component work is now in Spin and what needs to be done before we can support component imports in the manifest?